src/controllers/dynakube/istio/reconciler.go

```diff
@@ -60,15 +59,7 @@ func (reconciler *Reconciler) initializeIstioClient(config *rest.Config) (istioc
 // Reconcile - runs the istio's reconcile workflow,
 // creating/deleting VS & SE for external communications
 func (reconciler *Reconciler) Reconcile(instance *dynatracev1beta1.DynaKube, communicationHosts []dtclient.CommunicationHost) (bool, error) {
-	enabled, err := CheckIstioEnabled(reconciler.config)
-	if err != nil {
-		return false, fmt.Errorf("istio: failed to verify Istio availability: %w", err)
-	}
-	log.Info("istio: status", "enabled", enabled)
-
-	if !enabled {
-		return false, nil
-	}
+	log.Info("istio: reconciling")
```

Comment:
isn't this `istio: ` prefix outdated and should be replaced with a logger name?

Reply:
You are right, will remove the "istio: " prefix